### PR TITLE
Rspamd: better scores for mx_check symbols

### PR DESCRIPTION
**Pull request by marrco**
discussion on https://github.com/mailcow/mailcow-dockerized/issues/576

### CODE REVIEW

**Comment by andryyy** [comment]
@mkuron Can you merge it after a review? :)

**Comment by mkuron** [comment]
Yes, will do once I get a chance to test it on my server.

**Comment by mkuron** [comment]
Discussion still ongoing in #576, so this PR hasn't been merged yet. I'm currently leaning toward disabling the mx_check module completely.

**Comment by marrco** [comment]
I waited a few days to check more logs. The new proposed values seem fine to me and coherent with what those will likely be in the next rspamd release. The main issue was a low value for a timeout falsely returning non-connectable MX for servers that implement greet pause/greeting delay as an antispam measure.

File: data/conf/rspamd/local.d/mx_check.conf [hunk 1]
```diff
@@ -1,7 +1,8 @@
-timeout = 1.0;
+timeout = 8.0;
 symbol_bad_mx = "MX_INVALID";
 symbol_no_mx = "MX_MISSING";
 symbol_good_mx = "MX_GOOD";
+symbol_implicit_mx = "MX_IMPLICIT";
```
Comment (mkuron) [inline data/conf/rspamd/local.d/mx_check.conf]: Remove this. Mx-check module doesn't know about MX_IMPLICIT.

File: data/conf/rspamd/local.d/metrics.conf [hunk 2]
```diff
@@ -17,3 +17,25 @@ group "bayes" {
...
+	  score = 2.0;
+	  description = "No MX record";
+	  one_shot = "true";
+	}
+	symbol "MX_GOOD" {
+	  score = -0.01;
+	  description = "MX was ok";
+	  one_shot = "true";
+	}
+	symbol "MX_IMPLICIT" {
```
Comment (mkuron) [inline data/conf/rspamd/local.d/metrics.conf]: Remove this. Only internal symbols go here.

**Review by mkuron — DISMISSED** [review]

File: data/conf/rspamd/local.d/metrics.conf (hunk 2 above)
Comment (marrco) [reply data/conf/rspamd/local.d/metrics.conf]: ok, fixed

File: data/conf/rspamd/local.d/mx_check.conf (hunk 1 above)
Comment (marrco) [reply data/conf/rspamd/local.d/mx_check.conf]: ok, fixed

**Comment by mkuron** [comment]
Tested and merged. Thanks, @marrco, for your contribution.

I had to change the composite score as the rspamd documentation says
>you can define a composite that fires when two specific symbols are found and replace these symbols weights with its score

**Comment by marrco** [comment]
thank you for helping me @mkuron

**Comment by andryyy** [comment]
Thanks!

**Pull request merged**